src/BlazorUI/Bit.BlazorUI/Components/Navs/NavBar/BitNavBar.scss

```
@@ -50,12 +118,75 @@
+    font-size: var(--bit-nbr-ico-fs);
 }
 
+// The label is laid out as a single grid track holding the text, which is what lets the
+// HideUnselectedText mode collapse it: an fr track is animatable, where display: none is not.
 .bit-nbr-txt {
-    font-size: $tg-fs-sm;
+    display: grid;
+    min-width: 0;
+    max-width: 100%;
+    font-size: var(--bit-nbr-txt-fs);
+    // A label of a known line box is what lets the HideUnselectedText mode reserve the room it takes
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)